docs/about.md

```diff
@@ -1,5 +1,11 @@
-# About Sample Target
+# About MakeCode Arcade
 
-Example of editor target for [Microsoft MakeCode](https://makecode.com/).
+MakeCode Arcade is a code editor to build Retro Arcade games for the browser
+and handheld consoles.
 
-See [GitHub repo](https://github.com/Microsoft/pxt-arcade) for details.
+* [Try it now!](https://arcade.makecode.com)
+
+
+* [Projects](/projects)
```

Comment:
the projects link doesn't work for me when I 'view file' then click the link

Reply:
Links to other markdown file do not work from the GitHub view

Reply:
but none of them are links to markdown pages?

Reply:
Links the do not have an extension are implicitely markdown pages